suite-native/module-trading/src/components/buy/TradingProviderPicker.tsx

```
@@ -0,0 +1,50 @@
+};
+
+const notImplementedCallback = () => {
+    // eslint-disable-next-line no-console
+    console.log('Not implemented');
+};
+
+export const TradingProviderPicker = ({ form }: TradingProviderPickerProps) => {
+    const { translate } = useTranslate();
+    const providers = useSelector(selectTradingBuyProviders);
+
+    const provider = providers && providers[form.watch('provider')];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)